master/usage/configuration/index.md

```diff
@@ -1,11 +1,7 @@
 ---
-title: Configuring Calico
+title: Configuring Systems for use with Calico
```

Comment:
Thanks - I've created that doc and moved the OpenStack bits to it.

I renamed the remainder of the doc to "Configuring Conntrack"